### PR TITLE
Add 14x14_timer_add_outline update change log

### PR DESCRIPTION
**Pull request by EOSullivanBerlin**

### CODE REVIEW

File: icons/14x14_timer_add_outline.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (driesd) [inline icons/14x14_timer_add_outline.svg:1]:
```suggestion
<svg width="14" height="14" viewBox="0 0 14 14" xmlns="http://www.w3.org/2000/svg">
```

**Review by driesd — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,12 @@
 
 ### Dependency updates
 
+## [1.5.0] 2021-08-03
```
Comment (ArnaudWeyts) [inline CHANGELOG.md:15]: Please adhere to the changelog rules, usually we create a separate branch when making a release: https://github.com/teamleadercrm/ui-icons#how-to-make-a-release

**Review by ArnaudWeyts — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (EOSullivanBerlin) [reply CHANGELOG.md]: I will remove this and add another branch for the release.

**Review by lorgan3 — APPROVED** [review]

**Review by ArnaudWeyts — APPROVED** [review]

**Review by JorenSaeyTL — APPROVED** [review]

File: icons/14x14_timer_add_outline.svg [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<svg width="14" height="14" viewBox="0 0 14 14" xmlns="http://www.w3.org/2000/svg">
+<g opacity="0.84">
```
Comment (JorenSaeyTL) [inline icons/14x14_timer_add_outline.svg:2]: I think this opacity needs to be removed

File: icons/14x14_timer_add_outline.svg [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+<path d="M7 3C4.79137 3 3 4.79137 3 7C3 9.20935 4.79089 11 7 11V13C3.6868 13 1 10.3144 1 7C1 4.38804 2.66984 2.16538 5 1.34154V1C5 0.447715 5.44772 0 6 0H8C8.55228 0 9 0.447715 9 1V1.34137C11.3307 2.16508 13 4.38786 13 7H11C11 4.79089 9.20935 3 7 3Z" fill="#1A1C20"/>
+<path d="M7 4C7.55228 4 8 4.44772 8 5V7C8 7.55228 7.55228 8 7 8C6.44772 8 6 7.55228 6 7V5C6 4.44772 6.44772 4 7 4Z" fill="#1A1C20"/>
+<path d="M12 8V10H14V12H12V14H10V12H8V10H10V8H12Z" fill="#1A1C20"/>
```
Comment (JorenSaeyTL) [inline icons/14x14_timer_add_outline.svg:5]: The fill needs to be removed as well

**Review by JorenSaeyTL — CHANGES_REQUESTED** [review]

**Review by ArnaudWeyts — APPROVED** [review]

**Review by driesd — APPROVED** [review]

**Pull request merged**